analysis/subset_data/SubsetBam.wdl

```diff
@@ -0,0 +1,169 @@
+workflow SubsetBam {
+  # Bam to subset
+  File aligned_gold_bam
+
+  # How many cells to grab from the largest and smallest end
+  Int num_large_cells
+  Int num_small_cells
+
+  # How many reads a cell must have before it can be selected
+  Int? min_reads_per_cell
+
+  # Array of comma separated tag values to put in each fastq e.g ["ORIG", "SR", "CB,UB"] "ORIG" means to
```

Comment:
This is acutally just used to pick what quality tags to write in each of the bam/fastq files.  So for each bam/fastq you want to create, im asking what sequence tags go into bam_n, what quality tags go into bam_n.  If no quality tags are given then each base is given Q30 by default.  I just realized I dont give the option to not pass in quality tags and will change that to mimic how the tool currently works.

Reply:
Oh, got it. I think that `barcode` and `quality` tags are inextricably linked, as both are necessary to generate a fastq file (otherwise you'd have to create random qualities). 

Given this, the inputs should reflect this logic. If it were not a command-line tool, I would encapsulate these pairs in a 2-tuple. I'm not sure what the best way to do this on the command line would be. 

Perhaps a multi-input similar to how picard works, with one required input and then optional additional ones? 
```
barcode_and_quality_tags=CB,CY \
barcode_and_quality_tags=UB,UY # optional
```